src/go/wsl-helper/cmd/k3s_kubeconfig.go

```diff
@@ -53,6 +54,10 @@ type kubeConfig struct {
 	Extras map[string]interface{} `yaml:",inline"`
 }
 
+const (
```

Comment:
does it have to be a multi-line style? can it be a single-line declaration since there is only one const?